test.bats

```diff
@@ -141,3 +141,10 @@ teardown() {
     unzip .serverless/sls-py-req-test.zip -d puck
     ! ls puck/hug
 }
+
+@test "py3.6 can package flask with zip option and no explicit include" {
+    sed -i'.bak' -e 's/zip: *false/zip: true/' -e 's/include://' -e 's/^.*handler.py//' serverless.yml
```

Comment:
Apologies here - I spent 15 minutes trying to get the right sed syntax for multiline in one expression before smashing my keyboard and writing the second expression.